sections/main-collection-banner.liquid

```diff
@@ -1,5 +1,13 @@
 {{ 'component-collection-hero.css' | asset_url | stylesheet_tag }}
 
+{%- style -%}
+  @media screen and (max-width: 749px) {
+    .collection-hero--with-image .collection-hero__inner {
+      padding-bottom: calc({{ settings.media_shadow_vertical_offset | at_least: 0 }}px + 2rem);
+    }
+  }
+{%- endstyle -%}
```

Comment:
<img width="363" alt="Screen Shot 2022-01-12 at 11 51 35 AM" src="https://user-images.githubusercontent.com/26778637/149185035-4a6131a7-2fbd-4abe-9181-a0368b67791f.png">

I wonder if we should be trying to do something similar when the offset is negative. It can overlap the text a little bit.

Reply:
With both vertical offset set to the negative max and blur to the max it didn't mess with it so much 🤔  So I thought it should be fine.